src/parcsr_mv/par_vector.c

```diff
@@ -972,37 +974,59 @@ hypre_ParVectorPrintIJ( hypre_ParVector *vector,
    comm         = hypre_ParVectorComm(vector);
    global_size  = hypre_ParVectorGlobalSize(vector);
    partitioning = hypre_ParVectorPartitioning(vector);
-
-   /* multivector code not written yet */
-   hypre_assert( hypre_ParVectorNumVectors(vector) == 1 );
-   if ( hypre_ParVectorNumVectors(vector) != 1 ) { hypre_error_in_arg(1); }
+   local_vector = hypre_ParVectorLocalVector(vector);
+   local_size   = hypre_VectorSize(local_vector);
 
    hypre_MPI_Comm_rank(comm, &myid);
    hypre_MPI_Comm_size(comm, &num_procs);
 
    hypre_sprintf(new_filename, "%s.%05d", filename, myid);
-
    if ((file = fopen(new_filename, "w")) == NULL)
    {
-      hypre_error_w_msg(HYPRE_ERROR_GENERIC, "Error: can't open output file %s\n");
+      hypre_error_w_msg(HYPRE_ERROR_GENERIC, "Error: can't open output file!");
```

Comment:
Should we actually include the filename here in the print statement to be clear?

Reply:
Done in [ba9687f](https://github.com/hypre-space/hypre/pull/1033/commits/ba9687f24dbd6a1f25da04ed6d073634bc5a1c2e)